client/my-sites/stats/activity-log/rewind-alerts.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ThreatAlert from './threat-alert';
+import { getRewindAlerts } from 'state/selectors';
+
+export class RewindAlerts extends Component {
+	render() {
+		const { alerts: { threats }, translate } = this.props;
+
+		if ( ! threats.length ) {
+			return null;
+		}
+
+		return (
+			<Fragment>
+				<Card highlight="error">
+					{ translate( 'These items require your immediate attention' ) }
+				</Card>
+				{ threats.map( threat => <ThreatAlert key={ threat.signature } threat={ threat } /> ) }
```

Comment:
I'm concerned about sites that have several (or dozens of) threats. This does happen, and I'd hate for the entirety of their screen to fill with red alerts.

This is obviously part of the implementation we can discuss elsewhere, but I thought I would post a note here.

Reply:
Thanks. I'm not going to address this yet, especially since the design discussion is still open.